src/SignalR/specs/HubProtocol.md

```diff
@@ -31,6 +31,8 @@ In the SignalR protocol, the following types of messages can be sent:
 | `Completion`          | Callee         | Indicates a previous `Invocation` or `StreamInvocation` has completed. Contains an error if the invocation concluded with an error or the result of a non-streaming method invocation. The result will be absent for `void` methods. In case of streaming invocations no further `StreamItem` messages will be received. |
 | `CancelInvocation`    | Caller         | Sent by the client to cancel a streaming invocation on the server.                                                             |
 | `Ping`                | Caller, Callee | Sent by either party to check if the connection is active.                                                                     |
+| `StreamComplete`      | | |
+| `StreamData`          | | |
```

Comment:
I think I'm going to remove this change and keep https://github.com/aspnet/SignalR/issues/2619 for tracking it because we're considering changing the messages.